src/lib/Libutil/misc_utils.c

```diff
@@ -756,7 +756,7 @@ escape_delimiter(char *str, char *delim, char esc)
 
 	while (*str != '\0') {
 		/* We dont want to use strncmp function if delimiter is a character. */
-		if ((*str == esc && !(*(str + 1) == '\'' || *(str + 1) == '\"'))
+		if ((*str == esc && !(*(str + 1) == '\'' || *(str + 1) == '\"' || *(str + 1) == ','))
```

Comment:
As much as we are changing this if statement, It is becoming more and more difficult to read and very special case centric which is good.
If there is a way to refactor it then please do that.